app/helpers/application_helper.rb

```diff
@@ -10,4 +10,15 @@ def set_random_user_name()
     c = subsets.sample(length).join
     "username" + c
   end
+
+  STUFFTYPE = ["Good", "Bad", "Fix"]
```

Comment:
move these variables in the model

Reply:
Two models use this so, I have placed it in helper method and used .freeze with them,
Dynamic constant assignment error was showing , so it put it outside the functions